### PR TITLE
Sadoharu html css popup

### PR DESCRIPTION
**Pull request by Sadoharu**
### Hooli-style Popup

[Demo](https://sadoharu.github.io/html-css-popup) |
[Code base](https://github.com/Sadoharu/html-css-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by HannaSyn** [comment]
Great job! 
See my comments below

**Comment by HannaSyn** [comment]
Please, follow all the instructions from task, you missed some of them:
1. All interactive elements must be marked as such on hover - header menu items included
2. You should be able to switch between all interactive elements using only the keyboard (no mouse, don't forget the styles for the :focus state) - now you can only open/close menus

File: submissions/sadoharu/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    <link rel="icon" type="image/png" sizes="96x96" href="./favicon/favicon-96x96.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./favicon/favicon-16x16.png">
+    <link rel="manifest" href="./favicon/manifest.json">
+    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-TileImage" content="./favicon/ms-icon-144x144.png">
+    <meta name="theme-color" content="#ffffff">
+</head>
+<body>
+<header class="header">
+    <nav class="nav-bar">
```
Comment (HannaSyn) [inline submissions/sadoharu/html-css-popup/index.html:29]: Please, move `nav` tag to list with navigation, it is not necessary to include `logo` in it.

File: submissions/sadoharu/html-css-popup/style.css [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+@font-face {
+    font-family: 'icomoon';
+    src: url('./fonts/icomoon.woff2') format('woff2'),
+        url('./fonts/icomoon.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+    font-display: block;
+}
+
+
```
Comment (HannaSyn) [inline submissions/sadoharu/html-css-popup/style.css:30]: Please, remove unnecessary white-spaces, only 1 is required

File: submissions/sadoharu/html-css-popup/style.css [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+  text-transform: none;
+  line-height: 1;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+
+.icon-bell-solid:before {
+  content: "\e900";
+}
```
Comment (HannaSyn) [inline submissions/sadoharu/html-css-popup/style.css:45]: You should add white-space here =)

File: submissions/sadoharu/html-css-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+
+.nav-bar .logo h1 {
+    
+    font-weight: bold;
+    font-size: 20px;
+    margin-left: 60px;
+}
+
+
+
```
Comment (HannaSyn) [inline submissions/sadoharu/html-css-popup/style.css:110]: Please, remove unnecessary white-spaces, only 1 is required

File: submissions/sadoharu/html-css-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+.nav-bar .logo {
+    position: relative;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+
+    color: #613011;
+
+
```
Comment (HannaSyn) [inline submissions/sadoharu/html-css-popup/style.css:88]: Please, remove unnecessary white-spaces

**Review by HannaSyn — CHANGES_REQUESTED** [review]

**Comment by Sadoharu** [comment]
Please, review, again.

**Comment by HannaSyn** [comment]
> Please, review, again.

You did a great work! 
Let's fix one more small but important detail - when you tab through navigation, it would be great to have possibility after opening the popup menu, immediately tab to the first icon of the menu, and not go further along the navigation list in header. Also, after clicking on `Show more`, be able to continue following the list of icons, and only after the popup icons, continue navigation in the header.

File: submissions/sadoharu/html-css-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+<body>
+<header class="header">
+        <a tabindex="1" class="logo" href="#">
+            <img src="./img/coffee-capuccino-cat-emoji-remixit-cute-food-kawaii-backgrounds-1585244.png" alt="logo">
+            <h1>Sadoharu training page</h1>
+        </a>
+        <nav class="nav-bar">
+            <ul class="nav-bar_list">
+                <li class="nav-bar_list_item mail"><a tabindex="1" href="#">Mail</a></li>
+                <li class="nav-bar_list_item images"><a tabindex="1" href="#">Images</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/sadoharu/html-css-popup/index.html:36]: For this small page using `tabindex` attribute is redundant.
When user opened popup and presses the Tab key focus must go to the first element in popup In your case focus goes through the rest menu items and after all, items go inside popup. Remove `tabindex` attribute and see how behavior will change.

File: submissions/sadoharu/html-css-popup/style.css [hunk 7]
```diff
@@ -0,0 +1,298 @@
...
+    border-radius: 5px;
+}
+
+.nav-bar .nav-bar_list_item.popup-button input{
+    margin: 0;
+    height: 0;
+    width: 0;
+}
+
+.nav-bar .nav-bar_list_item.popup-button .popup-icon:checked ~ .popup {
```
Comment (A-Ostrovnyy) [inline submissions/sadoharu/html-css-popup/style.css:153]: Avoid styling elements using a too-specific CSS selector.
[Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) about specificity in CSS

File: submissions/sadoharu/html-css-popup/style.css [hunk 8]
```diff
@@ -0,0 +1,298 @@
...
+    outline: 1px solid #6130114f;
+    border-radius: 5px;
+}
+
+.nav-bar .nav-bar_list_item.popup-button .popup-icon:focus ~ .icon-grip-solid:before {
+    outline: 1px solid #6130114f;
+    border-radius: 5px;
+}
+
+.nav-bar .nav-bar_list_item.popup-button input{
```
Comment (A-Ostrovnyy) [inline submissions/sadoharu/html-css-popup/style.css:147]: Avoid styling by HTML tag. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good Work!

**Comment by Sadoharu** [comment]
okay. I fix this. pls review

**Review by HannaSyn — APPROVED** [review]

File: submissions/sadoharu/html-css-popup/index.html [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/blogs.png" alt="Blog">Blog</a></li>
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/calendar.png" alt="Calendar">Calendar</a></li>
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/chat.png" alt="Chat">Chat</a></li>
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/classroom.png" alt="Classrrom">Classrrom</a></li>
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/contacts.png" alt="Contacts">Contacts</a></li>
+                            <li class="popup-showmore">
+                                <input class="popup-more" type="checkbox" id="button">
+                                <label class="popup-lable" for="button"></label>
+                                <ul id="more" class="more" >
+                                    <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/documents.png" alt="Documents">Documents</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/sadoharu/html-css-popup/index.html:51]: Read about how to populate `alt` attribute in the right way.
Sorry for russian but it's a good [article](https://htmlacademy.ru/blog/articles/alt-text)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
You still have too specific CSS selectors, try to reduce their specificity.

**Comment by Sadoharu** [comment]
i tried to fix it, pls review

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
But for icons better an empty `alt` attribute)

**Comment by Sadoharu** [comment]
done

**Review by A-Ostrovnyy — APPROVED** [review]
Well done

**Pull request merged**